lab_1/main.py

```diff
@@ -34,18 +33,14 @@ def remove_stop_words(tokens: list, stop_words: list) -> list:
     stop_words = ['the', 'is']
     --> ['weather', 'sunny', 'man', 'happy']
     """
+    pass
 
-    def remove_stop_words():
-        tokens = str(input('Введите предложение ')).split()
-
-        stop_words = str(input('Введите стоп-слова ')).split()
-
-        final_list = [word for word in tokens if word not in stop_words]
+    with open("stop_words.txt") as file:  # конструкция with...as позволяет не закрывать файл вручную
```

Comment:
you shouldn't read any files in main.py
This is for start.py

stop_words is an argument of the remove_stop_words function
consider that it exists already